tendrl/node_agent/discovery/sds/plugins/discover_gluster_sds_details.py

```diff
@@ -50,14 +50,14 @@ def discover_storage_system(self):
         ret_val['detected_cluster_name'] = "gluster-%s" % cluster_id
         ret_val['peers'] = gfs_peers
 
-        # Check if the file /etc/tendrl/redhat-gluster-storage.json
+        # Check if the file /usr/share/glusterfs/release exists.
         # if the file exists, read the version details from this
-        if os.path.exists('/etc/tendrl/redhat-gluster-storage.json'):
+        if os.path.exists('/usr/share/glusterfs/release'):
```

Comment:
I thought you had finalized on `/etc/tendrl/redhat-gluster-storage.json`

Reply:
So as per BZ#1559507 now glusterfs itself would provide a file mentioning the version details. So we just need to read this file.

Check the comments history on BZ#1559507 for detailed discussion